build.sbt

```
@@ -22,7 +22,7 @@ val magnoliaVersion = "1.0.0-M4"
 val algebirdVersion = "0.13.9"
 val avroVersion = Option(sys.props("avro.version")).getOrElse("1.10.2")
 val bigqueryVersion = "v2-rev20210910-1.32.1"
-val bigtableVersion = "2.1.4"
+val bigtableVersion = "2.5.3"
```

```suggestion
val bigtableVersion = "2.5.1"
```

let's keep it in line with [Scio](https://github.com/spotify/scio/commit/154bf0be87da2c5b26ddcdb219e8dfede95d1244)/Beam